examples/object-initializer/object_initializer.description

```diff
@@ -1 +1,4 @@
-// In Ballerina, objects can have a constructor that can be used to initialize the object.
+// In Ballerina, objects have a method named `__init()` which is used to initialize the object when creating a
+// new object. There may be parameters as in any other method. However, the return type of the `__init()` method
+// should be a subtype of `error|()` which includes `()`. If an explicit definition for the `__init()` method
+// is not specified, there will be an implicit `__init()` method which does not accept any parameters and returns `()`.
```

Comment:
Do we need to mention the implicit `__init()` here? Isn't it like too much info?

Reply:
Added it for completeness.